inventory/group_vars/all/common.yml

```diff
@@ -99,18 +82,14 @@ controller_nodes:
 
 
 ### Messaging
-rabbit_host: 172.17.17.99
-rabbit_hosts: 172.17.17.70:5672,172.17.17.71:5672,172.17.17.72:5672
-rabbit_port: 5672
```

Comment:
rabbit_port was removed from there.  I can add in {{ rabbit_port | default (5672) }}